backend/go.mod

```diff
@@ -18,9 +18,11 @@ require (
 	github.com/golang-migrate/migrate/v4 v4.18.2
 	github.com/google/uuid v1.6.0
 	github.com/joho/godotenv v1.5.1
+	github.com/lestrrat-go/jwx/v3 v3.0.0-alpha3
```

Comment:
Using github.com/lestrrat-go/jwx which is a very complete JWx library, used by other large projects too (we had implemented in in Dapr too). In here I'm using it only for JWK support.